k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -895,10 +736,10 @@ function deployCortxSecrets()
     secrets="$(./parse_scripts/yaml_extract_block.sh "${solution_yaml}" "${yaml_content_path}" 2)"
 
     new_secret_gen_file="${secret_auto_gen_path}/${secret_fname}.yaml"
-    cp "${cfgmap_path}/templates/secret-template.yaml" "${new_secret_gen_file}"
+    cp "${cfgmap_path}/other/secret-template.yaml" "${new_secret_gen_file}"
```

Comment:
Note that `secret-template.yaml` is still the "old" template format, and so it can't reside in `templates/` or else Helm tries to evaluate it.